consent_manager/js_api/installation.rst

```diff
@@ -0,0 +1,61 @@
+Installation
+------------
+Consent Manager is fully integrated with Tag Manager. If you have already installed asynchronous snippet and you are using API only from Tag Manager tags, you are able use JavaScript API without aby pitfalls.
+
+Only one thing should be considered before using API is where you call commands. If your goal is to perform API method outside Tag Manager tags like in below example:
```

Comment:
```suggestion
The one thing that should be considered before using API is where you call commands. Lets assume that your goal is to perform API method outside Tag Manager tags like in below example:
```

Reply:
what do you think about changing order of `below example` to `example below` ?
```suggestion
The one thing that should be considered before using API is where you call commands. Lets assume that your goal is to perform API method outside Tag Manager tags like in example below:
```

Reply:
fixed